tests/unit/test_cache.py

```diff
@@ -55,6 +57,7 @@ def test_cache_bad_path(self) -> None:
         disk_path = "".join(random.choice(["x", "y"]) for _ in range(20))
         assert not _cache.exists(disk_path)
 
+    @pytest.mark.skipif(not pathlib.Path("README.md").is_file(), reason="No README.md")
```

Comment:
Where does this come from?

Reply:
The test itself does funny stuff with the README file (uploading it to the cloud, etc). We're implicitly assuming that we're running the tests from the root of the pycontrails directory (which isn't always the case).